src/main/java/com/aerospike/documentapi/data/DocumentFilterExp.java

```diff
@@ -0,0 +1,37 @@
+package com.aerospike.documentapi.data;
+
+import com.aerospike.client.exp.Exp;
+import com.aerospike.documentapi.util.DocumentExp;
+
+public class DocumentFilterExp {
+
+    private Exp exp;
```

Comment:
Can be final? since we are setting it in the constructor.

Reply:
It is initialized in constructor. The constructor could have accepted just an Exp as a parameter which could be less controllable. Currently there is an enum parameter, and Exp can be built using each of enum values given.